buildSrc/src/main/java/com/uber/okbuck/core/dependency/ExternalDependency.java

```diff
@@ -0,0 +1,95 @@
+package com.uber.okbuck.core.dependency;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+
+import java.io.File;
+
+/**
+ * Represents a pre packaged dependency from an external
+ * source like gradle/maven cache or the filesystem
+ */
+public final class ExternalDependency {
+
+    private static final String LOCAL_DEP_VERSION = "1.0.0";
```

Comment:
This is actually just an arbitrary version number we give to local dependencies i.e deps that are in vcs. It does not mean anything